app/js/components/labOrderEntry/PastOrders.jsx

```diff
@@ -14,9 +15,9 @@ const PastOrders = ({
       </thead>
       <tbody>
         {orders.map(order => (
-          <tr key={order.id}>
-            <td>{order.date}</td>
-            <td>{order.details}</td>
+          <tr key={order.uuid}>
```

Comment:
@Efosaok In cases like this, that's why the `shortid` package was installed, I would advise you use 
`shortid.generate()` so we don't have similar keys in any scenario.

Reply:
from experience, `shortid.generate()` is not usually efficient, it causes bugs if the component re-renders due to change in state, this will work fine as this is the specific id for that particular order that is coming from the database.

Reply:
I'm not familar with shortid, nor do I understand yet the intriacies of having a key and how React maps to the DOM, but I do agree with @Efosaok, the uuid of the order is guaranteed to be unique, so we wouldn't have to worry about that part of it.